packages/deployments/contracts/contracts/core/connext/helpers/Executor.sol

```diff
@@ -26,12 +28,35 @@ contract Executor is IExecutor {
 
   // ============ Properties =============
 
+  /**
+   * @notice Address of associated connext
+   */
   address private immutable connext;
+
+  /**
+   * @notice Properties set for access via reentrancy
+   * @dev Contracts that are being called during `execute` may set
```

Comment:
nit: Can we move this really great description of "properties" in general to be under the `// ============ Properties =============`